executor/metrics/metrics.go

```diff
@@ -17,21 +17,28 @@ package metrics
 import (
 	"time"
 
-	mgostatsd "github.com/scullxbones/mgo-statsd"
 	log "github.com/sirupsen/logrus"
 	"gopkg.in/mgo.v2"
 )
 
+// MetricsPusher is an interface for a metrics pusher
+type MetricsPusher interface {
```

Comment:
type name will be used as metrics.MetricsPusher by other packages, and that stutters; consider calling this Pusher